src/sonic-yang-models/yang-models/sonic-vlan-sub-interface.yang

```diff
@@ -70,6 +78,12 @@ module sonic-vlan-sub-interface {
                     description "Packet action when a packet ingress and gets routed on the same IP interface";
                     type stypes:loopback_action;
                 }
+
+                leaf vlan {
```

Comment:
Vlan or vlanid?
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-vlan.yang#L180-L184

Reply:
Hi @ganglyu, it is `vlan` per vlan sub intf HLD(https://github.com/sonic-net/SONiC/blob/master/doc/subport/sonic-sub-port-intf-hld.md)